docs/docusaurus/docs/gx_cloud/tutorials/getting_started/cloud_quickstart.md

```diff
@@ -0,0 +1,341 @@
+          "data_asset_name": "my_quickstart_table",
+      },
+  }],
+  "config_version": 1,
+  "class_name": "Checkpoint"
+}
+
+context.add_or_update_checkpoint(**checkpoint_config)
+checkpoint = context.get_checkpoint(checkpoint_name)
+print(checkpoint)
+
+
+# View results
+path_to_data = "https://raw.githubusercontent.com/great-expectations/gx_tutorials/main/data/yellow_tripdata_sample_2019-01.csv"
+df = pd.read_csv(path_to_data)
+
+batch_request = {
+    "runtime_parameters": {
+        "batch_data": df
+    },
+    "batch_identifiers": {
+        batch_id: "jan_2019"
+    }
+}
+
+result = context.run_checkpoint(ge_cloud_id=checkpoint.ge_cloud_id, batch_request=batch_request)
+print(result)
+```
+
+In the following steps we'll break down exactly what is happening here so that you can follow along and perform a <TechnicalTag tag="validation" text="Validation"/> yourself.
+
+## Steps
+
+### 1. Setup
+
+#### 1.1 Generate User Token
+
+Go to [“Settings” > “Tokens”](https://app.greatexpectations.io/tokens) in the navigation panel and generate a User Token. 
+These tokens are see-once and stored as a hash in Great Expectation Cloud's backend database. Once you copy API key and close the dialog, the Cloud UI will never show the token value again. 
+
+#### 1.2 Import modules
+
+:::tip 
+Any Python Interpreter or script file will work for the remaining steps in the guide, we recommend using a Jupyter Notebook, since they are included in the OSS GX installation and give the best experience of both composing a script file and running code in a live interpreter.
+:::
+
+Switch to Jupyter Notebook and import modules we're going to use in this tutorial.
+
+```python title="Jupyter Notebook"
+import great_expectations as gx
+import pandas as pd
+import os
+```
+
+#### 1.3 Create Data Context
+
+Paste this snippet into the next notebook cell to instantiate Cloud <TechnicalTag tag="data_context" text="Data Context"/>. 
+
+:::caution
+Please note that `cloud_access_token` is sensitive information and should not be committed to version control software. Alternatively, add these as [Data Context config variables](docs/guides/setup/configuring_data_contexts/how_to_configure_credentials.md)
+:::
+
+```python title="Jupyter Notebook"
+os.environ["GX_CLOUD_ORGANIZATION_ID"] = "<organization_id_from_the_app>"
+os.environ["GX_CLOUD_ACCESS_TOKEN"] = "<user_token_you_generated_in_the_app>"
+
+context = gx.get_context()
+```
+
+### 2. Connect to data
+
+Modify the following yaml code to connect to your <TechnicalTag tag="datasource" text="Datasource"/>
+
+:::caution
+Please note you should not include sensitive info/credentials directly in the config while connecting to your Datasource, since this would be persisted in plain text in the database and presented in Cloud UI. If credentials/full connection string is required, you should use [config variables file](docs/guides/setup/configuring_data_contexts/how_to_configure_credentials.md).
```

Comment:
```suggestion
Please note you should not include sensitive info/credentials directly in the config while connecting to your Datasource, since this would be persisted in plain text in the database and presented in Cloud UI. If credentials/full connection string are required, you should use a [config variables file](docs/guides/setup/configuring_data_contexts/how_to_configure_credentials.md).
```